[2024-12-23T19:59:17.223+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-23T19:59:17.253+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_connect.bash_ex1 scheduled__2024-12-22T08:00:00+00:00 [queued]>
[2024-12-23T19:59:17.261+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_connect.bash_ex1 scheduled__2024-12-22T08:00:00+00:00 [queued]>
[2024-12-23T19:59:17.262+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-23T19:59:17.277+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bash_ex1> on 2024-12-22 08:00:00+00:00
[2024-12-23T19:59:17.284+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1845) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-23T19:59:17.285+0000] {standard_task_runner.py:63} INFO - Started process 1852 to run task
[2024-12-23T19:59:17.285+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_connect', 'bash_ex1', 'scheduled__2024-12-22T08:00:00+00:00', '--job-id', '1055', '--raw', '--subdir', 'DAGS_FOLDER/dag_connect.py', '--cfg-path', '/tmp/tmpr6tde3qm']
[2024-12-23T19:59:17.286+0000] {standard_task_runner.py:91} INFO - Job 1055: Subtask bash_ex1
[2024-12-23T19:59:17.333+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_connect.bash_ex1 scheduled__2024-12-22T08:00:00+00:00 [running]> on host 87a1e11545b0
[2024-12-23T19:59:17.394+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_connect' AIRFLOW_CTX_TASK_ID='bash_ex1' AIRFLOW_CTX_EXECUTION_DATE='2024-12-22T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-22T08:00:00+00:00'
[2024-12-23T19:59:17.395+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-23T19:59:17.404+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-23T19:59:17.404+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Task 1"']
[2024-12-23T19:59:17.414+0000] {subprocess.py:86} INFO - Output:
[2024-12-23T19:59:17.415+0000] {subprocess.py:93} INFO - Task 1
[2024-12-23T19:59:17.415+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-23T19:59:17.416+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-23T19:59:17.433+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_connect, task_id=bash_ex1, run_id=scheduled__2024-12-22T08:00:00+00:00, execution_date=20241222T080000, start_date=20241223T195917, end_date=20241223T195917
[2024-12-23T19:59:17.460+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-23T19:59:17.477+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-23T19:59:17.479+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
